### ISSUE TITLE
ANCHORLINKS not working

### ISSUE THREAD

**Issue by stevenemrick** [jbake-org/jbake#222]
hi - I upgraded to pegdown 1.5.0 and I'm on Jbake 2.4.0 now.  In my jbake.properties file I want to use ANCHORLINKS in my markdown.extensions, as pegdown 1.5.0 now supports this.  But it's not working.  I need the functionality to create jump tags within a page.  Does anyone know why this is not working?   I also created a [ticket](https://github.com/sirthias/pegdown/issues/166) on the pegdown site.

**Comment by uli-heller** [comment]
I've tried to investigate a little bit  about about ANCHORLINKS:

* pegdown-1.5.0 has been released in Feb 2015
* on its [intro page](http://pegdown.org), I don't find anything related to anchorlinks
* within [github](https://github.com/sirthias/pegdown), I can search for ANCHORLINKS and find some references
* as mentioned by you, it has to show up within Extensions.java and it is there in the master branch
* within 1.5.0, I find it, too, but it has a bad value  0x300 vs. 0x400
* I guess we'll have to wait for pegdown-1.5.1, maybe it will work then

**Comment by uli-heller** [comment]
So, I made a few more tests:

1. I recompiled jbake-2.4.0 using pegdown-1.5.0 instead of pegdown-1.4.2
2. Within jbake.properties, I have this: `markdown.extensions=ALL,-HARDWRAPS`
3. Then, I "baked" some sample pages

With standard 2.4.0 using pegdown-1.4.2, the outcome looks like this:

```
<h2>Titel</h2><p>Als Titel...
```

With the freshly compiled 2.4.0 using pegdown-1.5.0, the outcome looks like this:

```
<h2><a href="#titel" name="titel">Titel</a></h2><p>Als Titel...
```

So I think:

* pegdown-1.5.0 allows you to use the anchorlinks when you specify "ALL" within your jbake.properties
* pegdown-1.5.1 will probably allow you to use ANCHORLINKS within your jbake.properties
